src/cljs/main/broadfcui/page/workspace/details.cljs

```diff
@@ -18,59 +18,39 @@
-              workspace
-              (when (get-in workspace [:workspace :realm])
-                [:div {:style {}}
-                 [:div {:style {:height 1 :backgroundColor "#bbb" :marginBottom 2}}]
-                 [:div {:style {:outlineTop (str "4px double #ccc")
-                                :backgroundColor "#ccc"
-                                :fontSize "small"
-                                :padding "4px 0"
-                                :textAlign "center"}}
-                  "This is a " [:b {} "restricted"] " workspace for TCGA Controlled Access Data."]])
-              :else
-              [:div {:style {:position "absolute" :marginTop "-1.5em"}}
-               [comps/Spinner {:height "1.5ex"}]])]))})
+     (let [{:keys [workspace]} props]
+       (when (and workspace (get-in workspace [:workspace :realm]))
+         [:div {:style {:paddingTop 2}}
+          [:div {:style {:backgroundColor "#ccc"
+                         :fontSize "small"
+                         :padding "4px 0"
+                         :textAlign "center"}}
+           "This is a " [:b {} "restricted"] " workspace for TCGA Controlled Access Data."]
+          [:div {:style {:height 1 :backgroundColor "#bbb" :marginTop 2}}]])))})
 
 (react/defc BucketBanner
   {:render
    (fn [{:keys [props]}]
-    [:div {:style {:position "relative"}}
-     (case (:bucket-access? props)
-       nil [:div {:style {:position "absolute" :marginTop "-1.5em"}}
-            [comps/Spinner {:height "1.5ex"}]]
-       true nil
-       false [:div {:style {}}
-             [:div {:style {:height 1 :backgroundColor "#bbb" :marginBottom 2}}]
-             [:div {:style {:outlineTop (str "4px double #ccc")
-                            :backgroundColor "#efdcd7"
-                            :fontSize "small"
-                            :padding "4px 0"
-                            :textAlign "center"}}
-              (cond (= 404 (:bucket-status-code props))
-                    (str "The Google bucket associated with this workspace"
-                         " does not exist. Please contact help@firecloud.org.")
-                    :else (str "The Google bucket associated with this workspace is currently unavailable."
-                         " This should be resolved shortly. If this persists for more than an hour,"
-                         " please contact help@firecloud.org."))]])])})
+     (when (= false (:bucket-access? props))
+       [:div {:style {:paddingTop 2}}
+        [:div {:style {:backgroundColor "#efdcd7"
+                       :fontSize "small"
+                       :padding "4px 0"
+                       :textAlign "center"}}
+         (cond (= 404 (:bucket-status-code props))
+           (str "The Google bucket associated with this workspace"
+                " does not exist. Please contact help@firecloud.org.")
+           :else (str "The Google bucket associated with this workspace is currently"
```

Comment:
Is this a place where clojure's built in multiline string concat would be sufficient?